feat(deb): add `priority` and `section` options (#126)
crabnebula-dev/cargo-packager: .changes/priority-and-section.md

---

> "cargo-packager": minor

Reviewer: ```suggestion
"cargo-packager": patch
"@crabnebula/packager": patch
```

---

> Add the support for priority and section in Debian Config

Reviewer: ```suggestion
Add `priority` and `section` options in Debian config
```
